### PR TITLE
Overlays: remove copyRootClassesToWrapper and hide elementAttr

### PR DESCRIPTION
**Pull request by Zedwag**

### CODE REVIEW

**Review by pomahtri — DISMISSED** [review]

File: packages/devextreme/js/ui/overlay.d.ts [hunk 1]
```diff
@@ -68,9 +61,9 @@ export interface dxOverlayOptions<TComponent> extends WidgetOptions<TComponent>
     deferRendering?: boolean;
     /**
      * @docid
-     * @deprecated dxOverlayOptions.wrapperAttr
+     * @deprecated
```
Comment (ksercs) [inline packages/devextreme/js/ui/overlay.d.ts:64]: don't we need to remove `deprecated` here?

**Comment by ksercs** [comment]
@Zedwag  I still see a using of `_ignoreElementAttrDeprecation` and `elementAttr` in `speed_dial_main_item.js` + `popupConfig.elementAttr` in `ui.dropDownEditor.js`. Please, recheck one more time that all occurences are removed.

**Review by ksercs — CHANGES_REQUESTED** [review]

File: packages/devextreme/js/ui/overlay.d.ts (hunk 1 above)
Comment (Zedwag) [reply packages/devextreme/js/ui/overlay.d.ts:64]: we discussed it in apiReview channel and dicided to keep @deprecated tag

**Comment by Zedwag** [comment]
@ksercs done

**Review by ksercs — APPROVED** [review]

**Review by alexanderPolosatov — APPROVED** [review]

**Review by vladaskorohodova — APPROVED** [review]

**Pull request merged**